### ISSUE TITLE
[Bug]: defStormsList error

### ISSUE THREAD

**Issue by thomaspibanez** [umr-amap/StormR#67]
### What happened?

Tried to extract all storms around a plot in Puerto Rico (-65.79781 18.2931 lon-lat) and got the following error message:

"Error in getStorm(s, name, season) : 
  No cyclone named KLAUS for season 1990
In addition: Warning message:
In convertLoi(loi) :
  longitude coordinate for loi has been corrected and set between 0 and 360 degree"

Strange because the cyclone names KLAUS for the season 1990 actually exists...

### Minimal Complete Verifiable Example

```R
NoAt<-StormR::defDatabase("IBTrACS.ALL.v04r00.nc",verbose=0,basin="NA")
EYNF<-StormR::defStormsList(NoAt,loi=c(-65.79780935896399,18.293103267852345))

sum(NoAt@database$names=="KLAUS"  & NoAt@database$seasons=="1990")
```

### Anything else we need to know?

Tied to remove the package and install a new version from GitHub but the error persists...

remove.packages("StormR")

devtools::install_github("umr-amap/StormR")

 

### Environment

<details>

</details>

**Comment by thomasarsouze** [comment]
This might be because "KLAUS" already exists in 1984 and defStormsLists doesn't handle this case correctly.
I'll check that more in details

**Comment by BaptisteDlp** [comment]
Normally the `getStorm(s, name, season)` call should handle the cases where several TCs share the same name, Maybe we should  check that before

**Comment by thomasarsouze** [comment]
This was indeed a `getStorm` issue that could not handle multiple Storms with the same name (always picking the first one).

Solved with 23df1a73c3ffdf0e95167ddcdff013338f895f86
